src/runtime_src/core/include/xrt/xrt_device.h

```diff
@@ -101,7 +101,8 @@ enum class device : unsigned int {
   aie_shim,
   dynamic_regions,
   vmr,
-  aie_mem
+  aie_mem,
+  aie_partitions
 };
```

Comment:
I am concerned with all these enumerators.  Are end-users really going to query the device for all these obscure enumerator values?   Or is this just for xbutil?

If latter, we should not expose in xrt:: APIs.

Reply:
I was thinking the same thing. Not sure if people would really want to get aie_partition info through xrt_devices.